app/models/work_type.rb

```diff
@@ -63,6 +61,8 @@ class InvalidType < StandardError; end
     'White paper', 'Working paper'
   ].freeze
 
+  MIXED_TYPES = (%w[Data Image Portfolio Music Software/Code Sound Text Video] + MORE_TYPES).uniq.sort.freeze
```

Comment:
Can we remove "Data", "Text" and "Video" from this list since they are already in `MORE_TYPES`?

Reply:
The screenshot doesn't display "Sound", should it?

Reply:
Removing dupes and `uniq`. Screenshot is from earlier code.